server/src/db/migrations/20230731193730_createActivities.cjs

```diff
@@ -0,0 +1,25 @@
+/**
+ * @typedef {import("knex")} Knex
+ */
+
+/**
+ * @param {Knex} knex
+ */
+exports.up = async (knex) => {
+    return knex.schema.createTable("activities", (table) => {
```

Comment:
looks like this feature is an index page of Categories, and Activities are not being used - this would be an example of "future proofing" the application by adding code to this feature that is not currently used. If Activities are the main entity of your app that users will be leaving Reviews about, I might suggest this first feature just work with Activities, and the Categories can be added in a later feature (in a new migration you can add the necessary reference column)